packages/apps/transfer/package.json

```diff
@@ -21,9 +21,11 @@
     "@kadena/chainweb-node-client": "workspace:*",
     "@kadena/client": "workspace:*",
     "@kadena/cryptography-utils": "workspace:*",
+    "@kadena/kadena-docs": "workspace:*",
     "@kadena/pactjs": "workspace:*",
     "@kadena/pactjs-cli": "workspace:*",
     "@kadena/react-components": "workspace:*",
+    "@kadena/react-ui": "workspace:*",
     "@radix-ui/colors": "~0.1.8",
     "@stitches/react": "1.3.1-1",
```

Comment:
Can't stitches be removed with the new layout?

Reply:
nope, we use old components also, until we remove all dependencies to old react components we need to keep them